### DISCUSSION TITLE
Who is "the community"?

### DISCUSSION THREAD

**Discussion by basvansambeek** [iota-community/Community-Governance#29] [General]
_This thread invites you to reflect on what you think is a good way of defining community membership, **without the obligation to provide a way to establish that with absolute certainty.**_ 

In order for us to establish an IOTA *Community* Treasury, maybe it's good to take a step back and somewhat formalize who we are. Or perhaps aren't.

Intuitively it's easy to refer to "the community", but who are we really? And where can you find us? When it comes to establishing who has to be included, how we reach out to to the rest of the group, and how we all get to vote securely and fairly on treasury matters, we need to define membership.

Having a definition will greatly help secure our community against malicious behavior from inside and outside, and ensure everybody who is relevant can be invited to participate.

**Comment by ebk1960** [comment]
The DAO Treasury community is made up of those who have a vested interest (financial and/or social) in the success of the IOTA ecosystem. This interest is manifested by community members' ownership of IOTA Tokens.

**Reply by basvansambeek** [reply]
So location wise, we can be found on the Tangle?

And does this imply there are token owners who are not in the treasury community?

**Reply by ebk1960** [reply]
Yes, its voluntary. They'll all get their invite, but its up to them to say 'yup, I'm in, where do I sign up'? Yes, they'll have to have a financial stake, with at least one IOTA token to be included.

**Reply by manunggal** [reply]
When we can expect to get the invitation? I'm in

**Reply by ghost** [reply]
This is a fact that annoys me very much. That everything is based on the possession of the token. There are some members in the community who are very active, e.g. some developers who don't hold a single IOTA. I, for example, keep only very small quantities in the low Mi area and only on a exchange. However, I would still count myself as an active member of the community.

To define a community member by whether he holds the token or not I think is a mistake and it excludes parts of the active community. But I must also admit that I unfortunately do not know a better solution. What annoys me even more.

And what would be the threshold anyway? Is 1 IOTA enough? 1 Gi? 

If there was the option, I would rather use some kind of DID and some sort of social points / activity points. e.g. high activity on Discord, active developer, etc.

**Reply by ebk1960** [reply]
Metz, 

I would suggest the holding of a single IOTA means you are qualified to vote. Just my personal opinion.

**Reply by ghost** [reply]
That then undermines the basic idea a bit. That would make absolutely everyone easely part of the community. Even people that never added anything to it or were never active. I don't think it can work without threshhold. if at all.

**Reply by ebk1960** [reply]
Well, you are then undermining your earlier argument about needing to possess IOTA tokens.

**Reply by ebk1960** [reply]
We have discussed badges for levels of activity/engagement in the DAO that could possibly be converted to NFT's or points that allow investment in DAO funded companies instead of investing in IOTA tokens. This would make it fairer on the community members who are not flush with IOTA but have shown a real commitment to the DAO/IOTA Ecosystem.

**Reply by ghost** [reply]
No, i still don't like that idea at all to use the Token. But if it is used, you would need a threshhold. Which opens another can of worms. But in general there has to be a better way to determine community membership that does not requires the ownership of the token.

**Reply by ebk1960** [reply]
> When we can expect to get the invitation? I'm in

That is the big question. It won't happen in June/July for sure.

**Reply by basvansambeek** [reply]
Before diving too deep into the how, Metz brings up an interesting point of what commitment entails.

I surely suspected there were people with few tokens and great commitment, and I’m happy Metz chose to speak out. It makes a weaker case for tokens to be the sole differentiator of commitment. Time is just as much a scarce good, if not scarcer than tokens.

So are there more ways people spend time on IOTA, without necessarily having tokens?

**Reply by Aconitin** [reply]
> This interest is manifested by community members' ownership of IOTA Tokens.

This is, in my opinion, probably one of the worst ways to claim membership of "the community". Somebody who spent weeks developing on the core protocol would be excluded in favor of somebody that bought his way in. I'm in favor of ignoring token ownership altogether here, though the question of course remains.

**Reply by basvansambeek** [reply]
Advocate of the devil: should membership of a *community* ***treasury*** be determined on merit, when in practice it could be an inflation to the token holders’ value?

**Reply by basvansambeek** [reply]
This is assuming a membership of the community treasury is a subset of the community, which is suggested. Is that so? And what are the implications?

**Reply by ebk1960** [reply]
Firstly, I'm not personally wedded to the idea of token ownership = community commitment. I just put that up to get reactions - which obviously seems to have worked - we've got the conversation started.

Bas also suggested looking at some tools that scrap contributions from Discord or commits on GitHub, etc., but does this capture all instances of community commitment? 

Personally, I'm all for allowing and weighting other types of commitment to the IOTA DAO Treasury or the wider ecosystem - if we can measure it.

**Reply by ebk1960** [reply]
Is it technically possible to link identifiable commitments to, say, badges that link to NFT's to reward folks for their efforts? I'm not a techie, so I'll leave it to better qualified folks to provide the answer(s) to that.

**Reply by basvansambeek** [reply]
I understand the desire to try and quantify commitment, but let’s try to observe it first. Who commits in any way or form, and how? 

Mods? Check. Devs? No doubt. But what about devs in private repo’s? And meme-makers? Even the bad ones? What about shit posters? What about alternative IOTA Discords whose members have been banned from the official one? How about regular posters with thoughtful comments on the machine economy on that hugely popular forum in South Korea that we’ve all never heard of until I brought it up randomly? Is following @IOTA on Twitter enough? Or should @IOTA_dev be followed too? Or do neither matter?

**Reply by sascha1337** [reply]
> This is a fact that annoys me very much. That everything is based on the possession of the token. There are some members in the community who are very active, e.g. some developers who don't hold a single IOTA. I, for example, keep only very small quantities in the low Mi area and only on a exchange. However, I would still count myself as an active member of the community.
> 
> To define a community member by whether he holds the token or not I think is a mistake and it excludes parts of the active community. But I must also admit that I unfortunately do not know a better solution. What annoys me even more.
> 
> And what would be the threshold anyway? Is 1 IOTA enough? 1 Gi?
> 
> If there was the option, I would rather use some kind of DID and some sort of social points / activity points. e.g. high activity on Discord, active developer, etc.

Thank you Metz, for representing me for example ~ currently i am only in possession of a few mi, even if i wish to hold more. Sometimes, for example, humans exchange the chance of getting rich with an investment, with having fiat available to survive, pay bills or spend the value instead for repairing a house from environmental damage _due to an earthquake_ ~ for the sake of the newborn child. 

There are priorities in reallife, that are specific for each person and cannot be generalized. Wallet balance can not be used as some equivalent to weight commitment and rights, to commit and contribute to the system - by any means. Me for example take insane risks and stress on my shoulders, simply for the sake of progress of the ecosystem. _'Komme was wolle.'_

So, my personal feeling is the following. 

- It should NEVER be the case, that someone that dedicates his life and full time and heart to the project, will be stripped in permissions, rights or opportunities, speaking of on-chain actions inside the iDAO system.

- Yes, there might be a few cases, where we can agree that there has to be throttlement in place, and that would be fraudulent activity, spam attacks, clearly malicious acting that is harmful for the system, or the spirit and common good of the project. 

- Also having MANA in mind ~ without SOME kind of ANY 'proof-of-engagement', virtual, but also **( !! ) in REALLIFE** ( #HansTheory ~ thread to be expanded s00n ), attached to your verifiable DID, the DAO CAN and SHOULD be scheduling  'restrictions' in terms of 'throttling' the speed, SCOPE or weight of effectiveness of influence.
       ​
-BUT, in return, i could still imagine some cases, where people with verifiable, positive proof-of-value / proof-of-engagement in the past, could kind of OPTIONALLY reflect their seriousness | confidence of some onchain action, if thats executing method XYZ, or submitting a proposal of funding, by making usage of our fancy UTXO address timelock feature. 

**OPTIONAL**. Just a thought. 

So you **effectively do not 8x** your voting rights via **staking | whaling** tokens for 1 year, **but** someone would have an _OPTION_, to make their claims publicly a bit more _trustworthy_ - because, if person X, throwing Y% or high-fiat-value Z into some locked pool and would propose some action, **the possibility of malicious intents for the protocol** in _combination_ of their DID reputation would logically be **less the case**, than some _zero-proof-of-engagement **DID_** interacting with 0 value-in-fiat tokens involved.

What you guys think? 

~ **Stay ZEN**, signed _Sascha1337_

**Reply by vromvonbeyond** [reply]
Maybe one way to have more weight on your opinion would be, if well known persons from the community would post their statements somewhere. So a website where you can easily see which developer supports which decision. People who have no idea what to vote for, could have a look at this website and see what respected people did vote for an act with their Token weight accordingly.

**Reply by ebk1960** [reply]
Sascha, great post.
